src/v/redpanda/admin/api-doc/security.def.json

```diff
@@ -21,4 +21,101 @@
             "type": "string"
         }
     }
+},
+"role_definition": {
+    "type": "object",
+    "properties": {
+        "role": {
+            "type": "string"
+        }
+    }
+},
+"role_description": {
+    "type": "object",
+    "properties": {
+        "name": {
+            "type": "string"
+        }
+    }
+},
+"role_member": {
+    "type": "object",
+    "properties": {
+        "name": {
+            "type": "string"
+        },
+        "principal_type": {
+            "type": "string"
+        }
+    }
+},
+"role_members_list": {
+    "type": "object",
+    "properties": {
+        "members": {
+            "type": "array",
+            "items": {
+                "type": "role_member"
+            }
+        }
+    }
+},
+"roles_list": {
+    "type": "object",
+    "properties": {
+        "roles": {
+            "type": "array",
+            "items": {
+                "type": "role_description"
+            }
+        }
+    }
+},
+"role": {
+    "type": "object",
+    "properties": {
+        "name": {
+            "type": "string"
+        },
+        "members": {
+            "type": "array",
+            "items": {
+                "type": "role_member"
+            }
+        }
+    }
+},
+"role_membership_update_response": {
+    "type": "object",
+    "properties": {
+        "role": {
+            "type": "string"
+        },
+        "added": {
+            "type": "array",
+            "items": {
+                "type": "role_member"
+            }
+        },
+        "removed": {
+            "type": "array",
+            "items": {
+                "type": "role_member"
+            }
+        },
+        "created": {
+            "type": "boolean"
+        }
+    }
+},
+"rbac_error_body": {
+    "type": "object",
+    "properties": {
+        "message": {
+            "type": "string"
+        },
+        "code": {
+            "type": "int"
```

Comment:
```suggestion
            "type": "integer"
```

Reply:
good catch

Reply:
actually I think `int` is correct 😅

Reply:
for posterity: `integer` is "correct" in the OpenAPI sense, but `json2code` prefers `int`. fun!